models/run_results.yml

```diff
@@ -1,6 +1,85 @@
 version: 2
 
 models:
+  - name: dbt_invocations
+    description: >
+      Attributes associated with each dbt invocation. Inserted at the end of each invocation.
+
+    columns:
+      - name: invocation_id
+        data_type: string
+        tests:
+          - not_null
+          - unique
+          - relationships:
+              field: invocation_id
+              to: ref('dbt_run_results')
+
+      - name: run_started_at
+        data_type: timestamp
+        description: Timestamp the invocation was started.
+
+      - name: run_completed_at
+        data_type: timestamp
+        description: Timestamp the invocation was completed
+
+      - name: generated_at
+        data_type: timestamp
+        description: Timestamp the invocation was completed and generated.
+
+      - name: command
+        data_type: string
+        description: dbt command that was used. For example, run.
+
+      - name: dbt_version
+        data_type: string
+        description: Version of dbt that was used in this invocation.
+
+      - name: elementary_version
+        data_type:
```

Comment:
Data type is string as well. Description could be - Version of the elementary package that was used in this invocation.